cmd/mole/main.go

```diff
@@ -102,6 +211,7 @@ func start(app cli.App) error {
 }
 
 func newAlias(app cli.App) error {
+	app.Detach = false
```

Comment:
Please take care of the invalid usage of `-detach` on `cli/cli.go`, function `Validate()`

Reply:
Oh I'm not sure about saving -detach in storage, so i think -detach should use in when -start
But now i got your point to save -detach in storage, i'm working on it